src/nidigital/metadata/functions.py

```diff
@@ -1448,6 +1460,7 @@
             },
             {
                 'direction': 'in',
+                'python_api_converter_name': 'convert_site_string',
```

Comment:
Where ever `convert_site_string` is used, you need to add
```
'type_in_documentation': 'str or int',
```
and change `type` to int.

Reply:
`'type_in_documentation': 'str or int',` added.

Can't change the type. The type is still `ViConstString` as far as the driver is concerned.